android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Node.kt

```diff
@@ -30,13 +42,15 @@ import java.util.concurrent.CopyOnWriteArrayList
 /**
  * Responsible for handling the addition and removal of child nodes.
  **/
+@SuppressWarnings("LargeClass")
```

Comment:
Suppress? Should we avoid having it?

Reply:
I don't see trivial choices how to do it right now, the class is still fairly simple. Maybe later yes.